extensions/positron-python/python_files/normalizeSelection.py

```diff
@@ -120,8 +120,12 @@ def normalize_lines(selection):
 
         # Insert a newline between each top-level statement, and append a newline to the selection.
         source = "\n".join(statements) + "\n"
+        # If selection ends with trailing dictionary or list, remove last unnecessary newline.
         if selection[-2] == "}" or selection[-2] == "]":
             source = source[:-1]
+        # If the selection contains trailing return dictionary, insert newline to trigger execute.
+        if check_end_with_return_dict(selection):
```

Comment:
We use this code so should double-check how this affects Cmd+Enter for lines like `return {}`. I doubt we've covered that in e2e tests.

Reply:
Just checked this out 👀 it looks like it will insert a newline in the script and in the console, but will still execute as expected.

https://github.com/user-attachments/assets/40b608ec-d199-4dc5-8201-0f06490b6985